### PR TITLE
04.HTML

### PR DESCRIPTION
**Pull request by ArtemUradau**
Could you please view my home task.
Thank you.

### CODE REVIEW

**Comment by ArtemUradau** [comment]
> Could you please view my home task.
> Thank you.

+05. HTML

File: auryadov/05.HTML/additional.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Форма регистрации</TITLE>
+	</HEAD>
+	<BODY>
+		<H1 align="center">Заполните анкету</H1>
+		<FORM ID="my_form1" name="pers_info" method="post" action="form.php">
+			<FIELDSET>
+				<LEGEND>Личные данные</LEGEND>
+				<INPUT type="text" name="surname" placeholder="Фамилия" pattern="^[А-Я] [а-я] {25}" required autofocus><BR>
```
Comment (pluhin) [inline auryadov/05.HTML/additional.html]: Need to remove white space form all pattern
ans 25- it equal 25 charset, but need not more that 25
Could you please align all pattern in this file
They do not work

File: auryadov/05.HTML/base.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+			</TR>
+			<TR>
+				<TD>
+					<H3>Отзыв:</H3>
+					<FORM method="post" action="http://sc.it-academy.by/re/form.php">
+						<FIELDSET>
+							<LEGEND>Оставьте ваш отзыв</LEGEND>
+							<INPUT type="text" name="login" required>
+							<SPAN>
+								<SUP>* </SUP>Имя
```
Comment (pluhin) [inline auryadov/05.HTML/base.html]: Where is label?

File: auryadov/05.HTML/base.html [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+							</SPAN><BR>
+							<INPUT type="password" name="password" required>
+							<SPAN>
+								<SUP>* </SUP>e-mail
```
Comment (pluhin) [inline auryadov/05.HTML/base.html]: Where is label?

File: auryadov/05.HTML/base.html [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+							</SPAN>
+						</FIELDSET>
+					</FORM>
+				</TD>
+				<TD>
+					<H3>Валидация данных HTML5:</H3>
+					<FORM method="post" action="http://sc.it-academy.by/re/form.php">
+						<FIELDSET>
+							<LEGEND>Строка поиска</LEGEND>
+							<INPUT type="text" name="number1"  pattern="^[А-Я] [а-я]"><BR>
```
Comment (pluhin) [inline auryadov/05.HTML/base.html]: Wrong pattern

File: auryadov/05.HTML/base.html [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+							<SPAN>
+								<I>Телефонный номер в формате<BR>+XXX-XX-XXXXXXX, где X - любое число</I>
+							</SPAN>
+							<INPUT type="number" name="number2" placeholder="+375-29-1234567" pattern="[0-9]"><BR>
```
Comment (pluhin) [inline auryadov/05.HTML/base.html]: Wrong pattern

**Comment by pluhin** [comment]
Any updates?

**Pull request merged**